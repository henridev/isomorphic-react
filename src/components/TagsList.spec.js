import React from 'react'
import {create} from "react-test-renderer";
import TagsList from './TagsList';

describe("Tags list", ()=>{
  it('Should render as expected', ()=>{
    const tree = create(<TagsList tags={['swift', 'css', 'html']}/>).toJSON()
    console.log('tree', tree)

    
    expect(tree).toMatchSnapshot();
  })
})

