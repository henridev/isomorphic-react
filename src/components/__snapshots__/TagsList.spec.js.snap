// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags list Should render as expected 1`] = `
<div>
  <code>
    swift
    ?
  </code>
  <code>
    css
    ?
  </code>
  <code>
    html
    ?
  </code>
</div>
`;
